final_installation/storage_server/102_create_user_homes.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+GROUPNAME=molecularbiodiversity
+for i in alk50uw wis26sq s216121
+do
+  ZFSDATASET=tank/"$GROUPNAME"/home/"$i"
+  sudo zfs create -o mountpoint=/new_home/"$i" "$ZFSDATASET"
+done
+
+GROUPNAME=supramolecularcellularsimulations
+for i in frm73du
+do
+  ZFSDATASET=tank/"$GROUPNAME"/home/"$i"
+  sudo zfs create -o mountpoint=/new_home/"$i" "$ZFSDATASET"
+done
+
+GROUPNAME=ecologicalmodeling
+for i in s216849 jus15cq
+do
+  ZFSDATASET=tank/"$GROUPNAME"/home/"$i"
+  sudo zfs create -o mountpoint=/new_home/"$i" "$ZFSDATASET"
+done
+
+GROUPNAME=evolutionarygenomics
+for i in ama42bf ark06eu jaf81qa s2046859
```

Comment:
s2046859 looks like a typo to me. I only know s numbers with 6 digits, this one has 7. Is this correct? !fix